microsoft/testsuites/xfstests/xfstesting.py

```diff
@@ -484,6 +486,17 @@ def xfstesting_azure_file_share_validation(
             # revert file into original status after testing.
             node.execute("cp -f /etc/fstab_cifs /etc/fstab", sudo=True)
 
+    def after_case(self, log: Logger, **kwargs: Any) -> None:
+        node: Node = kwargs.pop("node")
+        for path in [
+            "/dev/mapper/delay-test",
+            "/dev/mapper/huge-test",
+            "/dev/mapper/huge-test-zero",
+        ]:
+            if 0 == node.execute(f"ls -lt {path}", sudo=True).exit_code:
+                log.debug(f"dmsetup remove {path} to clean up env")
```

Comment:
This line is not necessary, the next line can output more details.

Reply:
remove this